### PR TITLE
move ssl info to co faq

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2816

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8a51246c63/how-to/). :bow:

File: content/rackspace-email/rackspace-email-faq.md [hunk 1]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+external MX servers), we will perform TLS if it is advertised by the
+remote server. When performing outgoing TLS, our servers are permissive
+with the certificate (in other words, if the site is using an untrusted
+or self-signed certificate, as long as it is a working certificate, we
+should still accept it).
+
+**Our outgoing SMTP servers will use TLS in an opportunistic fashion.**
+This means that our servers will attempt to open an SMTP transaction
+with the recipient server using TLS. If TLS cannot be successfully
+connected, the communication will default back to an unencrypted
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest "...defaults..."

File: content/rackspace-email/rackspace-email-faq.md [hunk 2]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+transmission of the data, also referred to as PLAINTEXT.
+
+Also our servers will respond to TLS and SSL requests to send mail to us encrypted.
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest removing the "Also" and "will" and making it "Our servers respond..."

File: content/rackspace-email/rackspace-email-faq.md [hunk 3]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+in and outbound email. For outgoing mail (any of our servers sending to
+external MX servers), we will perform TLS if it is advertised by the
+remote server. When performing outgoing TLS, our servers are permissive
+with the certificate (in other words, if the site is using an untrusted
+or self-signed certificate, as long as it is a working certificate, we
+should still accept it).
+
+**Our outgoing SMTP servers will use TLS in an opportunistic fashion.**
+This means that our servers will attempt to open an SMTP transaction
+with the recipient server using TLS. If TLS cannot be successfully
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest "by using"

File: content/rackspace-email/rackspace-email-faq.md [hunk 4]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+#### How do TLS and SSL work in Cloud Office?
+
+Transport Layer Security (TLS) and its predecessor, Secure Socket Layer
+(SSL), are cryptographic protocols that provide security for
+communications over networks. TLS and SSL encrypt the segments of
+network connections at the application layer to ensure secure end-to-end
+transit at the transport layer. For our purposes, they create an
+encrypted tunnel through which we send plain text emails.
+
+Cloud Office servers by default will attempt a TLS connection for both
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest "by default attempt"

File: content/rackspace-email/rackspace-email-faq.md [hunk 5]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+in and outbound email. For outgoing mail (any of our servers sending to
+external MX servers), we will perform TLS if it is advertised by the
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest removing "will."

File: content/rackspace-email/rackspace-email-faq.md [hunk 6]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+remote server. When performing outgoing TLS, our servers are permissive
+with the certificate (in other words, if the site is using an untrusted
+or self-signed certificate, as long as it is a working certificate, we
+should still accept it).
+
+**Our outgoing SMTP servers will use TLS in an opportunistic fashion.**
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest removing "will."

File: content/rackspace-email/rackspace-email-faq.md [hunk 7]
```diff
@@ -70,3 +70,28 @@ Log in to your control panel and click the **Support** menu at the top of the sc
...
+This means that our servers will attempt to open an SMTP transaction
```
Comment (RackerKate) [inline content/rackspace-email/rackspace-email-faq.md]: Suggest removing "will."

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9c8dc9d226/how-to/)

**Pull request merged**